code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -1125,7 +1125,7 @@
 
 	if(alien && alien == IS_VOX)
 		M.adjustOxyLoss(-2 * REM)
-		M.adjustToxLoss(-2 * REM)
+		M.adjustToxLoss(REM)
```

Comment:
I think you forgot a `-` there
This makes it *deal* toxin damage

Reply:
"Oops"